docker-entrypoint.sh

```diff
@@ -14,6 +14,11 @@ codegen="${cli}/target/openapi-generator-cli.jar"
 # list won't be considered a "real" command. We can get around that a bit by checking CLI completions beforehand if it exists.
 commands="list,generate,meta,help,config-help,validate,version"
 
+if [ $# == 0 ]; then
+	echo "Missing a command"
```

Comment:
@digininja thanks for the PR. Can we also list the commands in the message so that users know what commands to provide?